test/src/test/java/org/corfudb/integration/WorkflowIT.java

```diff
@@ -273,4 +276,27 @@ public void clusterResizingTest2() throws Exception {
 
         shutdownCorfuServer(p0);
     }
+
+    @Test
+    public void forceRemoveTwoDeadNodes() throws Exception {
```

Comment:
This is very similar to clusterResizingTest2. Can we merge them?

Reply:
And I think these 2 tests are different because they force remove different sets of nodes. We can leave both tests in that case.

Reply:
The problem our current tests didnt catch this case was because the force remove was sensitive to the node order in the layout.